Nimble.xcodeproj/xcshareddata/xcschemes/Nimble-macOS.xcscheme

```diff
@@ -16,7 +16,7 @@
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "1F925EAC195C0D6300ED456B"
                BuildableName = "Nimble.framework"
-               BlueprintName = "Nimble-OSX"
+               BlueprintName = "Nimble-macOS"
```

Comment:
The OS X (macOS) CI builds are broken now with the scheme name change. 😢 

- https://travis-ci.org/Quick/Nimble/jobs/149146818
- https://travis-ci.org/Quick/Nimble/jobs/149146824